### PR TITLE
Add startosis version of the module

### PR DESCRIPTION
**Pull request by gbouv**
The only thing that cannot be achieved right now is to pass in the number of services to start as params of the CLI command. But this is something that can easily be tweaked by updating the `PARAM` section of the script before running it

### CODE REVIEW

File: kurtosis-module/impl/datastore_army_kurtosis_module.go [hunk 1]
```diff
@@ -44,13 +44,14 @@ func (module *DatastoreArmyKurtosisModule) Execute(enclaveCtx *enclaves.EnclaveC
...
 	if err != nil {
-		return "", stacktrace.Propagate(err, "An error occurred serialzing the Lambda result object to JSON")
+		return "", stacktrace.Propagate(err, "An error occurred serializing the Lambda result object to JSON")
 	}
 	return string(resultJsonBytes), nil
 }
 
 // ====================================================================================================
-//                                       Private helper functions
+//
```
Comment (h4ck3rk3y) [inline kurtosis-module/impl/datastore_army_kurtosis_module.go:53]: gofmt screw up

File: startosis/datastore-army-module.star [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    service_config = struct(
+        container_image_name = datastore_image,
+        used_ports = {
+            datastore_port_id: struct(number = datastore_port_number, protocol = "TCP")
+        }
+    )
+    add_service(service_id = unique_service_id, service_config = service_config)
+
+def add_multiple_datastore_services(num_datastores):
+    for i in range(num_datastores):
```
Comment (h4ck3rk3y) [inline startosis/datastore-army-module.star:35]: nit: s/i/index/g

**Review by h4ck3rk3y — APPROVED** [review]
This looks alright but how does this get executed at the moment, you clone this and run the cli on it? Or do we run an interpreter that loads this? Or will the go sdk call this module?

**Comment by gbouv** [comment]
Once the [CLI PR is merged](https://github.com/kurtosis-tech/kurtosis/pull/166), you will be able to run it with a simple:
```
kurtosis startosis exec datastore-army-module.star [--enclave-is datastore-army-module]
```

**Review by h4ck3rk3y — CHANGES_REQUESTED** [review]
needs a CHANGELOG entry, preferably breaking

**Comment by h4ck3rk3y** [comment]
> Once the [CLI PR is merged](https://github.com/kurtosis-tech/kurtosis/pull/166), you will be able to run it with a simple:
> 
> ```
> kurtosis startosis exec datastore-army-module.star [--enclave-is datastore-army-module]
> ```

Oh so you clone it and then do it. Just UX doesn't have to be a part of this at all but what about something like kurtosis startosis exec github.com/path/to/main.star, that loads and executes the main function?

File: startosis/datastore-army-module.star [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+#vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv SCRIPT vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv#
+
+def add_datastore_service(unique_service_id):
+    print("Adding service " + unique_service_id)
+
+    service_config = struct(
+        container_image_name = datastore_image,
+        used_ports = {
+            datastore_port_id: struct(number = datastore_port_number, protocol = "TCP")
```
Comment (mieubrisse) [inline startosis/datastore-army-module.star:29]: Magic `TCP` constant

**Review by mieubrisse — APPROVED** [review]

File: startosis/datastore-army-module.star [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+num_datastores = 3
+
+#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ END OF PARAMETERS ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^#
+
+
+#vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv STATIC CONST vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv#
+
+module_name_for_logging = "datastore_army_module"
+
+datastore_image = "kurtosistech/example-datastore-server"
```
Comment (mieubrisse) [inline startosis/datastore-army-module.star:12]: Recommend using UPPER_SNAKE_CASE convention for constants (will be a good habit to get into as we get bigger scripts)

File: startosis/datastore-army-module.star (hunk 3 above)
Comment (mieubrisse) [reply startosis/datastore-army-module.star:29]: ideally this is an enum that Starlark easily provides (don't have to do for now though)

Comment (gbouv) [reply startosis/datastore-army-module.star:29]: Yes we can interpret Starlark scripts "in an environment", in which we can put all sort of stuffs. We should start thinking of what a good "Startosis base environment" looks like. Alternatively, we can use modules.

**Comment by gbouv** [comment]
@h4ck3rk3y yes we could add the execution of a script directly from GH, that'd be a FR :P
Added changelog, though I think Breaking is a bit too much here, we're not breaking anything of the current behaviour. I added a Feature for now. Once/When we remove the Go module code, we can do Breaking

**Review by mieubrisse — APPROVED** [review]
Good for me

**Review by h4ck3rk3y — APPROVED** [review]
Yalla @gbouv

**Pull request merged**